eng/pipelines/aggregate-reports.yml

```diff
@@ -39,7 +39,7 @@ stages:
           displayName: 'Build All Libraries'
           inputs:
             mavenPomFile: pom.xml
-            options: '$(DefaultOptions) -T 2C -DskipTests -Dgpg.skip -Dmaven.javadoc.skip=true -Dcheckstyle.skip=true -Dspotbugs.skip=true -Djacoco.skip=true -Drevapi.skip=true -Paggregatereports'
+            options: '$(DefaultOptions) -T 2C -DskipTests -Dgpg.skip -Dmaven.javadoc.skip=true -Dcodesnippet.skip=true -Dcheckstyle.skip=true -Dspotbugs.skip=true -Djacoco.skip=true -Drevapi.skip=true -Paggregatereports'
```

Comment:
These option lists are starting to look pretty long and duplicated. We should consider sharing those in a global property.